sdk/servicebus/service-bus/src/clientEntityContext.ts

```diff
@@ -123,26 +118,10 @@ export namespace ClientEntityContext {
       isClosed: false,
       requestResponseLockedMessages: new ConcurrentExpiringMap<string>(),
       isSessionEnabled: !!options.isSessionEnabled,
-      messageSessions: {},
-      expiredMessageSessions: {}
+      messageSessions: {}
     };
 
     (entityContext as ClientEntityContext).getReceiver = (name: string, sessionId?: string) => {
-      if (sessionId != undefined && entityContext.expiredMessageSessions[sessionId]) {
```

Comment:
`getReceiver` would now return `undefined` instead of throwing an error.

Reply:
Hmmm, we are using the `getReceiver` in renewlock and message settlement. renewlock is not relevant for sessions,
message settlement properly throws the `SessionLockLostError` when the receiver is undefined. So, everything seems fine with the change.

Reply:
Oh, I just realized you've mentioned the error message too in the description!